armsrc/appmain.c

```diff
@@ -881,12 +886,16 @@ static void PacketReceived(PacketCommandNG *packet) {
         }
         case CMD_LF_SNIFF_RAW_ADC: {
             struct p {
-                uint32_t samples : 31;
-                bool     verbose : 1;
+                uint32_t samples  : 30;
```

Comment:
May I use `uint8_t` there?

Reply:
we use the bool in structs ( or rather @doegox  introduced it )  since it was supported on the oldest compiler version we support.
it gives use clarify of code  which we like.

Reply:
If so I'll keep using `bool`